test/e2e/autoscaling/OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
```

Comment:
I don't have place to comment, so commenting here.
This won't work out of the box - you need to also modify:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/e2e_test.go#L25